internal/generator/suite.go

```diff
@@ -191,3 +193,68 @@ func (s *Suite) String() string {
 
 	return spaceRegex.ReplaceAllString(strings.TrimSpace(result.String()), "\n")
 }
+
+const bashSuiteTemplate = `
+function setup() {
+	{{ .Setup }}
+}
+
+function cleanup() {
+	{{ .Cleanup }}
+}
+`
+
+// BashString generates bash script for the suite
+func (s *Suite) BashString() string {
+	setup := strings.Join(s.getCompleteSetup(), "\n")
+	cleanup := strings.Join(s.getCompleteCleanup(), "\n")
+
+	tmpl, err := template.New("test").Parse(bashSuiteTemplate)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	var result = new(strings.Builder)
+
+	_ = tmpl.Execute(result, struct {
+		Dir     string
+		Cleanup string
+		Setup   string
+	}{
+		Dir:     s.Dir,
+		Cleanup: cleanup,
+		Setup:   setup,
+	})
+	for _, test := range s.Tests {
+		result.WriteString(test.BashString())
+	}
+	result.WriteString("\n")
+	result.WriteString("setup\n")
+	for _, test := range s.Tests {
+		result.WriteString("test" + test.Name + "\n")
+	}
+	result.WriteString("cleanup\n")
```

Comment:
The main reason we want to generate bash scripts for tests is to allow us to execute arbitrary bash commands between setup and cleanup, without modifying files in deployments repo.

Instead of always executing `setup; test; cleanup` please add some way to execute individual functions.

For example, I think it would be convenient to use script args. Like this:
```bash
./some/path/suite.sh setup
some-custom-command
./some/path/suite.sh cleanup
```